internal/app/release.go

```diff
@@ -292,21 +292,21 @@ func getChartVersion(chart, version string) (string, string) {
 }
 
 // testRelease creates a Helm command to test a particular release.
-func (r *release) test(p *plan) {
+func (r *release) test(afterCommands *[]command) {
```

Comment:
I don't think this needs to be a pointer here, slices are references to arrays.

Reply:
Actually I did that on explicit as I'm changing the slice size causing it to allocate it as a new one. When tested, going with no pointers gives no results in the slice after function test() has been run, hence pointers